pydantic_core/core_schema.py

```diff
@@ -30,10 +30,11 @@ class CoreConfig(TypedDict, total=False):
     config_choose_priority: int
     # if configs are merged, which should take precedence, default 0, default means child takes precedence
     config_merge_priority: int
-    # settings related to typed_dicts only
+    # used for types with fields -- models, dataclasses, typed dicts
     extra_fields_behavior: ExtraBehavior
+    # related to typed_dicts only
     typed_dict_total: bool  # default: True
-    # used on typed-dicts and tagged union keys
+    # used for models, dataclasses, and tagged union keys
```

Comment:
noticed these seemed out of date, not sure if I updated it correctly, feel free to correct

Reply:
i think total does only apply to `typed_dict`.

Reply:
I realize this little preview window is a bit confusing, that change there applies to the following line, not the preceding. And there are a couple other suggestions. (Maybe I'm the one misunderstanding here though..)

Reply:
ok